frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesList.cy.ts

```diff
@@ -66,9 +94,629 @@ describe('PipelinesList', () => {
+    viewPipelineServerModal.shouldHaveSecretKey('sdsd');
+    viewPipelineServerModal.shouldHaveEndPoint('https://s3.amazonaws.com');
+    viewPipelineServerModal.shouldHaveBucketName('test-pipelines-bucket');
+
+    viewPipelineServerModal.findDoneButton().click();
+  });
+
+  it('should delete pipeline server', () => {
+    initIntercepts();
+
+    cy.interceptK8sList(
+      DataSciencePipelineApplicationModel,
+      mockK8sResourceList([mockDataSciencePipelineApplicationK8sResource({})]),
+    );
+
+    cy.interceptK8s(
+      DataSciencePipelineApplicationModel,
+      mockDataSciencePipelineApplicationK8sResource({ dspaSecretName: 'aws-connection-test' }),
+    );
+
+    cy.interceptK8s(
+      'DELETE',
+      SecretModel,
+      mockSecretK8sResource({ name: 'ds-pipeline-config', namespace: projectName }),
+    ).as('deletePipelineConfig');
+    cy.interceptK8s(
+      'DELETE',
+      SecretModel,
+      mockSecretK8sResource({ name: 'pipelines-db-password', namespace: projectName }),
+    ).as('deletePipelineDBPassword');
+    cy.interceptK8s(
+      'DELETE',
+      SecretModel,
+      mockSecretK8sResource({ name: 'dashboard-dspa-secret', namespace: projectName }),
+    ).as('deleteDSPASecret');
+    cy.interceptK8s(
+      'DELETE',
+      DataSciencePipelineApplicationModel,
+      mockDataSciencePipelineApplicationK8sResource({ namespace: projectName }),
+    ).as('deleteDSPA');
+
+    projectDetails.visitSection(projectName, 'pipelines-projects');
+
+    pipelinesGlobal.selectPipelineServerAction('Delete pipeline server');
+    deleteModal.findSubmitButton().should('be.disabled');
+    deleteModal.findInput().fill('Test Project pipeline server');
+    deleteModal.findSubmitButton().should('be.enabled').click();
+
+    cy.wait('@deletePipelineDBPassword');
+    cy.wait('@deletePipelineConfig');
+    cy.wait('@deleteDSPASecret');
+    cy.wait('@deleteDSPA');
+  });
+
+  it('imports a new pipeline', () => {
+    initIntercepts();
+
+    const uploadPipelineParams = {
+      display_name: 'New pipeline',
+      description: 'New pipeline description',
+    };
+    const uploadedMockPipeline = buildMockPipelineV2(uploadPipelineParams);
+
+    // Intercept upload/re-fetch of pipelines
+    pipelineImportModal.mockUploadPipeline(uploadPipelineParams, projectName).as('uploadPipeline');
+    pipelinesTable
+      .mockGetPipelines([initialMockPipeline, uploadedMockPipeline], projectName)
+      .as('refreshPipelines');
+
+    cy.interceptK8sList(
+      DataSciencePipelineApplicationModel,
+      mockK8sResourceList([mockDataSciencePipelineApplicationK8sResource({})]),
+    );
+
+    cy.interceptK8s(
+      DataSciencePipelineApplicationModel,
+      mockDataSciencePipelineApplicationK8sResource({ dspaSecretName: 'aws-connection-test' }),
+    );
+
+    projectDetails.visitSection(projectName, 'pipelines-projects');
+
+    // Wait for the pipelines table to load
+    pipelinesTable.find();
```

Comment:
Can you add assertion for this ?
```suggestion
    pipelinesTable.find().should('exist');
```

Reply:
FYI `.should('exist')` is not necessary however it is good practice just to show the intent when other devs read the code.